en/docs/observe/mi-observe/working-with-management-api.md

```diff
@@ -1519,6 +1810,34 @@ The management API has multiple resources to provide information regarding the d
 	</endpoint>
   	```
 
+	??? note "Search for a Registry Resource by name"
```

Comment:
It would be better if we can mention here that, it will recursively search for the file/directory containing given searchKey in its name, in current and subsequent subfolders as well

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/6433/commits/5f196449167f0d69798c32ca17b2b61470553d0b